### ISSUE TITLE
Birthday message sent even to the person whose birthday has passed

### ISSUE THREAD

**Issue by kryptokinght** [thealphadollar/salvator#31]
If a person's birthday was a day before, fb's birthday page(https://www.facebook.com/events/birthdays/) even shows the birthdays that were one day ago. 
The getRawBirthdayData() function reads these birthdays as well.

**Comment by thealphadollar** [comment]
I don't think this happens, I would like to see an example; never happened to me!

**Comment by kryptokinght** [comment]
![bug](https://user-images.githubusercontent.com/21311242/50606505-01fded00-0eec-11e9-9f3d-c5bf2148dbb7.png)
This is the screenshot of the Birthday's page. The guy **Arnab Mondal** had his birthday the previous day. As you can see in the message box a birthday message has already been sent the other day. But another message was sent on 00:05 by the cron job which I scheduled to run at 00:05.
I guess the error is because the recent birthdays box looks exactly similar to a birthday box. So salvator scrapes the birthday from the recent birthdays box as well!
